.github/workflows/main.yml

```diff
@@ -91,6 +91,14 @@ jobs:
         path: |
           dist/*.whl
 
+  imports:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: pip install isort pycln pyopengl
```

Comment:
Think it doesn't require pyopengl installation everytime in the CI - can be added to the ignore list when there are even many internal imports? This one little demo anyway does a * star import, so doesn't help, and the demo doesn't change in eons.

Reply:
It's for pycln to be able to analyse it (because it does a star import). But I can also add it to pycln ignores to not have to import it in CI if the CI job stays.